.github/workflows/ci.yml

```diff
@@ -7,6 +7,8 @@ on:
   pull_request:
     branches:
       - master
+  schedule:
```

Comment:
what is scheduled?

Reply:
Currently the automated tests run when something is pushed to a PR or main branch. By adding this schedule, they will also run every week, even if there are no commits. The aim here is to ensure that even if the underlying libraries release new versions, TextWiser is still installable and tests still pass. If one of the dependencies breaks TW, this would tell us.

Reply:
awesome! Running in "schedule" seems like a great practice for continuous testing. 

Can you please bring this up with the team, so that one person in each other library can mimic the same? So we not only learn and improve but also scale it out.

Great addition!